support-frontend/assets/components/footerCompliant/FooterWithPromoTerms.tsx

```diff
@@ -0,0 +1,199 @@
+import type { ReactNode } from 'react';
```

Comment:
this file is `DigitalFooter.tsx` renamed and refactored. Both DS and GW use this file now. It enables T&Cs for 2 offers on a single page